migrator.sh

```diff
@@ -347,6 +355,54 @@ decode_auth() {
   fi
 }
 
+##
+# Get all the tags for a name at the target repository.
+#
+# If MIGRATE_ALL is set to true, the list will not be queried but always be empty
+#
+# Usage: query_tags_to_skip <image_name>
+# Return: json-list of all tags in the form ["latest", "dev", "123"]. If the request fails, the list will be empty
+##
+query_tags_to_skip() {
+  IMAGE="${1}"
+
+  if [ ${SKIP_EXISTING_TAGS} = "false" ]; then
```

Comment:
${SKIP_EXISTING_TAGS} should have quotes, just in case of somehow someone putting in spaces